scripts/geopmpy/agent.py

```diff
@@ -171,10 +171,12 @@ class AgentConf(object):
 
     """
     def __init__(self, path, agent='monitor', options=dict()):
-        supported_agents = {'monitor', 'power_governor', 'power_balancer',
-                            'frequency_map'}
+        agent_name_cstr = gffi.new("char[]", 1024)
+        agents = []
+        while _dl.geopm_agent_name(len(agents), 1024, agent_name_cstr) >= 0:
+            agents.append(gffi.string(agent_name_cstr).decode())
```

Comment:
Use the `names()` function defined on line 129 instead of calling into gffi directly.  I think it'll look like:
```agents = names()```
That should be it.